pom.xml

```diff
@@ -63,6 +63,11 @@
             <artifactId>annotations</artifactId>
             <version>23.0.0</version>
         </dependency>
+        <dependency>
```

Comment:
It is better to use 
```
    <dependency>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-aop</artifactId>
    </dependency>
```
because in this case we guarantee right version  of the dependency.